server/database/config/connection.ts

```diff
@@ -1 +1,25 @@
- 
+import { Pool } from 'pg';
+import * as env2 from 'env2';
+
+const {
+  NODE_ENV, DB_URL_DEV, DB_URL_TEST, DATABASE_URL,
+} = process.env;
+
+let dbUrl: string;
+
+if (NODE_ENV === 'development') {
```

Comment:
I think u can use switch instead of duplicated condition

Reply:
Okay, Nada